src/app/public/modules/vertical-tabset/vertical-tabset.service.spec.ts

```diff
@@ -82,6 +82,27 @@ describe('Vertical tabset service', () => {
     service.addTab(tab1);
     service.addTab(tab2);
 
-    expect(service.activeTabContent()).toBe(undefined);
+    expect(service.activeTab()).toBe(undefined);
+  });
+
+  it('destroyed tab removes it from the service', () => {
```

Comment:
I think maybe this was supposed to be `destroy tab` instead of `destroyed tab`?

Reply:
`destroy tab` sounds better since that's the name of the function.  I'll fix this.